### PR TITLE
New package: gcr4-4.0.0

### PR DESCRIPTION
**Pull request by oreo639**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/gcr4/template [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+make_check_pre="dbus-run-session"
+
+# Package build options
+build_options="gir"
+build_options_default="gir"
+
+post_patch() {
+	vsed -e "s/gcr_soversion = .*/gcr_soversion = '0'/" -i meson.build
+	vsed -e "s/gck_soversion = .*/gck_soversion = '0'/" -i meson.build
+}
```
Comment (paper42) [inline srcpkgs/gcr4/template:29]: this was just for testing, it can e dropped

Comment (oreo639) [reply srcpkgs/gcr4/template]: Alright.

File: srcpkgs/gcr4/template [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+build_style=meson
+build_helper="gir"
+configure_args="$(vopt_bool gir introspection) $(vopt_bool gir gtk_doc)"
+hostmakedepends="gettext glib-devel gnupg gi-docgen gettext pkg-config openssh
+ libxslt $(vopt_if gir vala)"
+makedepends="libgcrypt-devel libsecret-devel p11-kit-devel libxslt-devel gtk4-devel"
+depends="desktop-file-utils hicolor-icon-theme gnupg"
+checkdepends="dbus"
+short_desc="GNOME crypto package"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (JamiKettunen) [inline srcpkgs/gcr4/template:14]: Don't know if this is normal but to me it looks a bit weird pushing new packages that are already orphaned ^^

Comment (paper42) [reply srcpkgs/gcr4/template:14]: I adapted this template from gcr which was orphaned. @oreo639 could you assign me as the maintainer?

Comment (oreo639) [reply srcpkgs/gcr4/template]: Sure.

**Pull request merged**